src/normalize.css

```diff
@@ -0,0 +1,129 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
```

Comment:
This is great, I'm curious how you came across CSS resets. One question though, you've named this file normalize.css, but I don't see you actually using https://github.com/necolas/normalize.css. Is this intentional?

Reply:
While working on styling, I noticed more margin than I had specified and upon inspection saw it was coming from a `user-agent-stylesheet`. That's when I realized the browser applies some default styling. I recalled I ran across this problem in the past on a personal project and used the CSS to remove the default styling. To avoid confusion, I'll go ahead and rename the file to `reset-css.css`

Reply:
So cool that you realized that and found a solution. That's the sort of thing I love to see! 👏🏻. I will say that normalize.css is kind of the modern equivalent of css resets so it might be worth to switch it up in a small PR in the future.